### PR TITLE
[final] fix subscriber attributes error codes

### PR DESCRIPTION
**Pull request by aboedo**
* updated invalid subscriber attributes error code: 7262 -> 7263
* added invalid subscriber attributes request body error code: 7264
* added a couple of error codes cases missing from a switch statement

✅  tested and working with a local khepri instance, forcing the error codes to appear on every request for test purposes.

### CODE REVIEW

File: Purchases/Public/RCPurchasesErrors.h [hunk 1]
```diff
@@ -75,7 +75,8 @@ typedef NS_ENUM(NSInteger, RCBackendErrorCode) {
     RCBackendPlayStoreGenericError = 7231,
     RCBackendUserIneligibleForPromoOffer = 7232,
     RCBackendInvalidAppleSubscriptionKey = 7234,
-    RCBackendInvalidSubscriberAttributes = 7262
+    RCBackendInvalidSubscriberAttributes = 7263,
+    RCBackendInvalidSubscriberAttributesBody = 7264
```
Comment (vegaro) [inline Purchases/Public/RCPurchasesErrors.h]: hmm I need to add that one to Android

File: Purchases/Public/RCPurchasesErrorUtils.m [hunk 2]
```diff
@@ -121,6 +121,10 @@
             return @"INVALID_APPLE_SUBSCRIPTION_KEY";
         case RCIneligibleError:
             return @"INELIGIBLE_ERROR";
+        case RCInsufficientPermissionsError:
+            return @"INSUFFICIENT_PERMISSIONS_ERROR";
+        case RCPaymentPendingError:
+            return @"payment_pending_error";
```
Comment (vegaro) [inline Purchases/Public/RCPurchasesErrorUtils.m]: uppercase, just to be consistent

Comment (aboedo) [reply Purchases/Public/RCPurchasesErrorUtils.m]: you know, I was convinced that I'd done this in uppercase. fixing.

**Comment by vegaro** [comment]
🚢

**Pull request merged**